buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -125,9 +125,9 @@ SPECS = ['ppc64_aix' : CURRENT_RELEASES,
          'x86-64_windows_cm': CURRENT_RELEASES,
          'x86-64_windows_xl' : CURRENT_RELEASES,
          'x86-64_windows_xl_cm': CURRENT_RELEASES,
-         'aarch64_linux' : ['8', '11'],
+         'aarch64_linux' : CURRENT_RELEASES,
```

Comment:
Please update the comments on lines 45-46 to match.

Reply:
Updated.